src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -141,6 +146,9 @@ public RegistrationStatusResponse migrateAccounts(List<MigrationAccount> account
         if (this.rpc == null) {
```

Comment:
I think it can check a condition using a rpcUrl.

Reply:
I agreed with that. 
I'll remove checking logic whether rpc is null or not.